engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1332,6 +1345,23 @@ private void translateBlockStatement(Tree tree, Collection<Expression> appendTo)
       default -> {
         var expressionStatement = translateExpression(tree);
         if (expressionStatement != null) {
+          int last = appendTo.size() - 1;
+          if (last >= 0
+              && appendTo.get(last) instanceof IfThenElse ife
+              && ife.falseBranchOrNull() == null) {
+            if (expressionStatement instanceof Application.Prefix app
+                && app.arguments().length() == 1
+                && app.function() instanceof Name.Literal lit
+                && "else".equals(lit.name())) {
+              var newIfe =
+                  IfThenElse.builder(ife)
+                      .falseBranchOrNull(app.arguments().apply(0).value())
+                      .build();
+              appendTo.set(last, newIfe);
+              // no appendTo.add, but return
+              return;
+            }
+          }
```

Comment:
This will not work correctly. This implementation is attempting to reinterpret a multiline call to function "else" as if `else` were a syntax that accepts a block, but it is not so simple; the arguments to a multiline call are *not statements*, so for example use of the `=` operator will not be parsed as an assignment statement.